audio_service/darwin/Classes/AudioServicePlugin.m

```diff
@@ -289,17 +293,40 @@ - (void) updateNowPlayingInfo {
             nowPlayingInfo[MPMediaItemPropertyArtist] = mediaItem[@"artist"];
         }
         if (mediaItem[@"duration"] != [NSNull null]) {
-            nowPlayingInfo[MPMediaItemPropertyPlaybackDuration] = [NSNumber numberWithLongLong: ([mediaItem[@"duration"] longLongValue] / 1000)];
+            nowPlayingInfo[MPMediaItemPropertyPlaybackDuration] = [NSNumber numberWithUnsignedLong: ([mediaItem[@"duration"] unsignedLongValue] / 1000)];
         }
         if (@available(iOS 3.0, macOS 10.13.2, *)) {
             if (artwork) {
                 nowPlayingInfo[MPMediaItemPropertyArtwork] = artwork;
             }
         }
-        nowPlayingInfo[MPNowPlayingInfoPropertyElapsedPlaybackTime] = [NSNumber numberWithInt:([position intValue] / 1000)];
     }
+
+    MPNowPlayingInfoCenter *center = [MPNowPlayingInfoCenter defaultCenter];
+    nowPlayingInfo[MPNowPlayingInfoPropertyMediaType] = @(MPNowPlayingInfoMediaTypeAudio);
     nowPlayingInfo[MPNowPlayingInfoPropertyPlaybackRate] = playing ? speed: [NSNumber numberWithDouble: 0.0];
-    [MPNowPlayingInfoCenter defaultCenter].nowPlayingInfo = nowPlayingInfo;
+    nowPlayingInfo[MPNowPlayingInfoPropertyElapsedPlaybackTime] = [NSNumber numberWithUnsignedLong:([position unsignedLongValue] / 1000)];
```

Comment:
Probably need to look at this datatype as well. It's native format is a signed double, not an unsigned long.